homeassistant/components/danfoss_air.py

```diff
@@ -0,0 +1,89 @@
+"""
+Support for Danfoss Air HRV.
```

Comment:
Move this file to `homeassistant/components/danfoss_air/__init__.py`. This allows all of the platforms to live alongside the component.

Reply:
Did not know this was possible. Makes it much easer to get an overview of what platforms a component uses.

Reply:
Yep, it's very new and makes things much cleaner!